transport.go

```diff
@@ -211,6 +211,15 @@ func getRequestFromEvent(event *Event, dsn *Dsn) (r *http.Request, err error) {
 		if r != nil {
 			r.Header.Set("User-Agent", fmt.Sprintf("%s/%s", event.Sdk.Name, event.Sdk.Version))
 			r.Header.Set("Content-Type", "application/x-sentry-envelope")
+
+			auth := fmt.Sprintf("Sentry sentry_version=%s, "+
+				"sentry_client=%s/%s, sentry_key=%s", apiVersion, event.Sdk.Name, event.Sdk.Version, dsn.publicKey)
```

Comment:
I removed `sentry_timestamp`, according to https://develop.sentry.dev/sdk/overview/#authentication

> This key is effectively deprecated, and it is ignored on the receiving side. Use the [sent_at envelope header](https://develop.sentry.dev/sdk/envelopes/#envelope-headers) instead.